main.go

```diff
@@ -97,6 +115,9 @@ func pointStream(csc <-chan []simpartsim.Coords) (etherdream.PointStream, error)
 		for cs := range csc {
 			dumpInPointStream(w, cs)
 		}
+		for {
```

Comment:
This is an animation loop, it never ends until you close the program. Ideally the simulation would just keep running instead of having a finite number of frames.